src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -107,6 +118,14 @@ export default {
 		setIsTextMoreThanOneLine() {
 			this.isTextMoreThanOneLine = this.$refs.absenceMessage?.scrollHeight > this.$refs.absenceMessage?.clientHeight
 		},
+
+		async openConversationWithReplacementUser() {
+			const conversation = await this.$store.dispatch('createOneToOneConversation', this.userAbsence.replacementUserId)
+			this.$router.push({
+				name: 'conversation',
+				params: { token: conversation.token },
+			}).catch(err => console.debug(`Error while pushing the new conversation's route: ${err}`))
+		},
```

Comment:
We don't need to manually create conversation here in the store, it is supported via `callUser` links. So you can push to `root` with `{ query: { callUser: userid } }`

Reply:
Making `this.$store.dispatch` is not possible for true links anyway if we want to make it a valid link.